microsetta_admin/server.py

```diff
@@ -227,6 +227,85 @@ def manage_projects():
                            result=result), 200
 
 
+@app.route('/email_stats', methods=['GET', 'POST'])
+def email_stats():
+    if request.method == 'GET':
+        project = request.args.get('project', None)
+        email = request.args.get('email')
+        if email is None:
+            # They want to search for emails, show them the search dialog
+            return render_template("email_stats_pulldown.html",
+                                   **build_login_variables(),
+                                   resource=None,
+                                   search_error=None)
+        emails = [email]
+    elif request.method == 'POST':
+        project = request.form.get('project', None)
+        emails, upload_err = upload_util.parse_request_csv_col(
+            request,
+            'file',
+            'email'
+        )
+        if upload_err is not None:
+            return render_template('email_stats_pulldown.html',
+                                   **build_login_variables(),
+                                   resource=None,
+                                   search_error=[{'error': upload_err}])
+    else:
+        raise BadRequest()
+
+    if project is str:
```

Comment:
I'm not sure I understand this. Do you mean `if isinstance(project, str)`? If so, I would recommend instead doing `if project is not None`